src/sys/socket/consts.rs

```diff
@@ -1,75 +1,253 @@
 pub use self::os::*;
 
-#[cfg(any(target_os = "linux", target_os = "android"))]
 mod os {
```

Comment:
I don't think we need this `os` module anymore since all constants are now included individually.